macros/PGbasicmacros.pl

```diff
@@ -2756,7 +2756,7 @@ sub begintable {
 		|| $displayMode eq 'HTML_asciimath'
 		|| $displayMode eq 'HTML_LaTeXMathML'
 		|| $displayMode eq 'HTML_img') {
-		$out .= "<TABLE BORDER='1' STYLE='text-align:center;'>\n"
+		$out .= '<table class="pg-table">'
```

Comment:
missing semicolon -- wouldn't perltidy complain about this?

Reply:
Yeah, probably.  It wasn't there before. I didn't notice that.  I will add it.

Reply:
Thanks for noticing that.  By the way, perltidy would automatically add that semicolon!